distribution/pom.xml

```diff
@@ -67,6 +68,58 @@
     </build>
 
     <profiles>
+        <!-- FIXME (DDF-1198) - Remove this profile when this pom inherits from the latest ddf-parent pom -->
```

Comment:
I do not understand why you are temporarily adding owasp as part of this PR.  I would suggest removing the oswasp commit from this PR and owasp will be added as part of the other pending PR.